app/views/layouts/application.html.erb

```diff
@@ -13,8 +13,8 @@
   <%= favicon_link_tag asset_pack_path('media/images/favicon.svg'), sizes: 'any' %>
   <%= favicon_link_tag asset_pack_path('media/images/govuk-icon-mask.svg'), rel: 'mask-icon', type: 'image/svg', color: "#0b0c0c" %>
   <%= favicon_link_tag asset_pack_path('media/images/govuk-icon-180.png'), rel: 'apple-touch-icon', type: 'image/png' %>
-  <%= stylesheet_pack_tag 'application', 'data-turbolinks-track': 'reload', media: 'all' %>
```

Comment:
This has never been plumbed together properly and is copy/pasted from a DfE app by Octavian a long time ago